src/lambda/dynamoStreamhandler/index.js

```diff
@@ -0,0 +1,103 @@
+import { default as sgMail } from '@sendgrid/mail'
+import * as qr from 'qrcode'
+import * as crypto from 'crypto'
+
+sgMail.setApiKey(process.env.SENDGRID_API_KEY)
+
+const sendConfirmationEmail = async (data) => {
+  const msg = {
+    to: data.email,
+    from: 'registration@mail.kuberoke.love',
+    subject: 'Your registration for Kuberoke',
+    text: 'You have successfully registered for Kuberoke!',
+    html: '<p>You have successfully registered for Kuberoke!</p>'
```

Comment:
Need to make these configurable.

Reply:
Idea would be to create one or more files for this in the respective environment folder which are then loaded and passed into the TF module.